app/assets/stylesheets/components/_phone-input.scss

```diff
@@ -1,8 +1,27 @@
 lg-phone-input {
   display: block;
 
+  .iti {
+    margin-top: units(1);
+  }
```

Comment:
The reason for this change is that `.usa-input` adds a default top margin of `0.5rem`, so instead of adding `margin-bottom-1` to our hint usage, we anticipate `.usa-input`'s margin. `intl-tel-input` unsets the margin, however, applying a style of `margin-top: 0 !important;` to the input element. Adding back the margin here restores that intended margin.